cmd/snap/cmd_snap_op.go

```diff
@@ -285,6 +310,7 @@ func refreshAll() error {
 
 	names := make([]string, len(updates))
 	for i, update := range updates {
+		// TODO: set JailMode from snapstate
```

Comment:
but this doesn't belong here, no? it would need to be in snapstate or api itself